Guidelines.md

```diff
@@ -83,18 +83,23 @@ The [Cake.Recipe] scripts support building [Wyam] projects and the created websi
 
 For details see [How to create gh-pages branch].
 
-## Integration on Cake website
-
-Addins published to nuget.org should be listed on the cakebuild.net website. This is especially relevant 
-to the audit process because the YAML files are used to build the list of known addins to be audited. As 
-of this writing, the audit process also gathers additional known addins from the addins listed in the
-"Addins" section of [Status.md](https://github.com/cake-contrib/Home/blob/master/Status.md#addins) in 
-the cake-contrib GitHub repository but be aware that this manually maintained list will eventually be 
-abandoned when we feel confident that most (if not all) addin author have created a YAML file for their
-addins.
-
-Follow the instructions [here](https://github.com/cake-build/website/blob/develop/README.md#addins) to 
-create a YAML file for your addin.
+## Deprecating an addin
+
+From time to time, a previously published addin  needs to be deprecated. This could be for a number of reasons:
+
+- It was created in error.
+- It has been superseded by another package.
+- It is an older package that no longer follows the guidelines.
+- Its package id has been changed to something that better fits with the package naming guidelines.
+
+All versions of this package could simply be unlisted from nuget.org, meaning that they could no longer be installed, however, this solution is not ideal. Any build script referencing this package would no longer works which is far from ideal. When a package needs to be deprecated, it needs to be handled in such a way that existing users will continue to be able to use the old package id, but take advantage of the replacement package, if there is one.
```

Comment:
The sentence about unlisting is not correct like this. Build script should continue to work even if package is unlisted, as unlisting only changes how the package appears on nuget.org but it still can be found. See [here](https://docs.microsoft.com/en-us/vsts/package/nuget/unlist-delete?view=vsts).

Reply:
Can you propose a better sentence?

Reply:
The only thing which speaks against unlisting packages IMHO is that it will break build scripts which don't have addins pinned (on the other hand there is not a single reason why one should do that...). Otherwise unlisting is the proposed way for nuget packge how to handle such scenarios. The difference to Chocolaty package here is that Chocolatey packages add a final version with a dependency to allow a seamless upgrade path

Reply:
One shouldn't reference an addin without pinning it, I agree but... we all know that people do that all the time!!! Anyway, can you help me come up with a more accurate sentence that let addin author know that we think they should leave a listed version of their addin with an updated title (which is important because the audit tool uses the presence of this prefix to determine that a given addin is deprecated).

Reply:
How about something like:

>All versions of this package could simply be unlisted from nuget.org, meaning that they could no longer be found, however, this solution is not ideal. Any build script referencing this package without a specific version would no longer works which is far from ideal. When a package needs to be deprecated, it needs to be handled in such a way that existing users will continue to be able to use the old package id, but take advantage of the replacement package, if there is one.